lib/log.js

```diff
@@ -39,7 +39,7 @@ function sanitise(raw) {
     raw = sanitiseConfigVariable(raw, 'JIRA_USERNAME');
   }
 
-  if (config.JIRA_USERNAME) {
```

Comment:
are sure this is the right thing? I'm not familiar with the config for the broker though (eg is USERNAME actually a misnomer here?)